package.json

```diff
@@ -2,6 +2,7 @@
   "name": "quoting-service",
   "description": "Quoting Service hosted by a scheme",
   "license": "Apache-2.0",
+  "version": "8.6.2-snapshot",
```

Comment:
Two versions here, delete this line or the next? You may also like to increment the version- though I'm not sure what version would be appropriate right now.

Reply:
@msk- here's the guidance regarding versioning: https://github.com/mojaloop/documentation/blob/master/contributors-guide/standards/versioning.md.. It would be 8.7.0-snapshot for now